android/app/src/test/java/network/mysterium/ui/TokenModelTest.kt

```diff
@@ -0,0 +1,22 @@
+package network.mysterium.ui
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class TokenModelTest {
+
+    @Test
+    fun testDisplayValueFormatting() {
+        var model = TokenModel(value = 124_00_000_000L)
```

Comment:
How does this compile? `value` is double. Did you mean to assign `token` property?